pkg/jwt/jwtToken.go

```diff
@@ -0,0 +1,62 @@
+package jwt
+
+import (
+	"encoding/base64"
+	"encoding/pem"
+	"fmt"
+	"time"
+
+	"github.com/golang-jwt/jwt/v5"
+)
+
+type JwtIssuer struct {
+	issuer string
+	secret string
+	ttl    time.Time
```

Comment:
my bad, let's do ttl(time to live) as time.Duration, as same as Claims.ExpiresAt

Reply:
https://pkg.go.dev/github.com/golang-jwt/jwt/v5#RegisteredClaims  data type of ``ExpiresAt`` accept ``time.Time`` .